.github/workflows/release.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Setting composite action runtime
         uses: actions/setup-node@v4
         with:
-          node-version: 16
+          node-version: 20
```

Comment:
let's use the same approach we use in the CI workflow for this (meaning, use the .nvmrc file)

Reply:
we already have .nvmrc file in root folder with lts/*, should I just remove the line 27 & 28?